### ISSUE TITLE
[Layer dialog] Connect style component to default style in layer

### ISSUE THREAD

**Issue by gino-m** [google/ground-platform#227]

**Comment by gino-m** [comment]
Hi @parulraheja98, care to take this one on? You'll also need to load the selected layer into the dialog and update the project when the user clicks save. Wdyt?

**Comment by parulraheja98** [comment]
Hi @gino-m , I can take this issue :).
Just had a question regarding it:
Will the Dialog have same fields like react app i.e. Default Style, Untitled Form etc.

**Comment by gino-m** [comment]
> Hi @gino-m , I can take this issue :).
> Just had a question regarding it:
> Will the Dialog have same fields like react app i.e. Default Style, Untitled Form etc.

Cool, thanks! It will be very similar, see screenshot in #224.

Some of the model objects are already implemented (Layer, Style, etc.). For reference on the database schema check out https://github.com/google/ground-platform/wiki/Cloud-Firestore-Representation.

**Comment by gino-m** [comment]
@DaoyuT FYI Parul is connecting the style editor to the layer dialog. Please be sure to coordinate when you get to #196.

**Comment by parulraheja98** [comment]
@gino-m , just a follow up question, I need to add to the frontend of dialog the name of the layer, color of the layer and  the form layout with ability to add more questions?

**Comment by gino-m** [comment]
> do I need to add to the frontend of dialog the ability to add more questions (#223)

I discussed this one with @DaoyuT and we agree the form editor is an easily separable piece of work that we'd love for you to take on. I'll follow up on details in separate issues.

> will the edit of default style change the color of the pin in the layers list?

Yes, it will. The side panel is being implemented by @gyarill who I presume will implement the markers that react to updates to the project model.

> Also one more question that is there any design of layers mock available as I see it was WIP as mentioned in #188

I added some mocks to related issues. I'll do another pass to make sure all the relevant ones are there before assigning them. Thanks!
